frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingDialogs/createDrawingDialog.component.tsx

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Not a big deal, but other v5 modals mostly use the name "modal" instead of "dialog"